app/src/main/java/org/akvo/flow/api/FlowApi.java

```diff
@@ -161,5 +150,8 @@ private String getTimestamp() {
         String TIMESTAMP    = "ts=";
         String LAST_UPDATED = "lastUpdateTime=";
         String HMAC         = "h=";
+        String VERSION      = "ver=";
+        String DEVICE_ID    = "devId=";
+        String ANDROID_ID   = "androidId=";
```

Comment:
`ver` and `devId` are already being read by the servlet (I just refactored the definition). But indeed, `androidId` is the new parameter that will need to be parsed. Might be wise to document this somewhere.